acceptance/bundle/deployment/bind/schema/script

```diff
@@ -0,0 +1,33 @@
+title "Bind schema test: "
+
+title "Substitute variables in the template: "
+export BUNDLE_UUID=$(uuid)
+export SCHEMA_NAME="test-schema-$(uuid)"
+envsubst < databricks.yml > out.yml && mv out.yml databricks.yml
+
+title "Create a pre-defined schema: "
+CATALOG_NAME=main
+$CLI schemas create ${SCHEMA_NAME} ${CATALOG_NAME} | jq '{full_name, catalog_name, comment}'
+
+title "Bind schema: "
+$CLI bundle deployment bind schema1 ${CATALOG_NAME}.${SCHEMA_NAME} --auto-approve
+
+title "Deploy bundle: "
+$CLI bundle deploy --force-lock --auto-approve
+
+title "Read the pre-defined schema: "
+$CLI schemas get ${CATALOG_NAME}.${SCHEMA_NAME}  | jq '{full_name, catalog_name, comment}'
+
+title "Unbind the schema: "
+$CLI bundle deployment unbind schema1
+
+title "Destroy the bundle: "
+$CLI bundle destroy --auto-approve
```

Comment:
All the cleanup steps will only be executed if test succeeds but if not, workspace will get a lot of hanging schemas. Maybe we could move it to script.cleanup? @denik any other ideas maybe?

Reply:
I agree script.cleanup is the right place. 

I'm not sure if it's executed in case of an error though, it might be not. 

@anton-107 can you check that and maybe patch acceptance_test.go to execute script.cleanup in separate processes regardless of status of the main script?

Reply:
I'll move the cleanup step to script.cleanup. I suggest to do the refactoring for the acceptance_test in a follow up pr

Reply:
+1 to follow up PR with trap. Also, please take a look you can add acceptance/selftest/trap to test that it works both in failed and successful exits